.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/app-prototype
+    docker:
+      - image: circleci/ruby:2.4.2-node-browsers
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: postgres
+          RAILS_ENV: test
+      - image: circleci/postgres:9.6.5-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: app_prototype_test
+          POSTGRES_PASSWORD: ""
+    steps:
+      - checkout
+
+      # Restore bundle cache
+      - type: cache-restore
+        key: bundle-{{ arch }}
+
+      # Bundle install dependencies
+      - run: bundle install --path vendor/bundle --jobs=4
+      - run: bundle clean --force
+
+      # Store bundle cache
+      - type: cache-save
+        key: bundle-{{ arch }}
+        paths:
+          - vendor/bundle
+
+      # Database setup
+      - run: bundle exec rake db:create
+      - run: bundle exec rake db:schema:load
```

Comment:
`db:setup` does the same as `db:create` + `db:schema:load` + `db:seed`, which saves several seconds vs running them separately.

Reply:
I like it... though I thought that we recently saw something that suggested the test db wasn't created or something like that when running db:setup. Ring a bell?

Reply:
I'm not sure, but our project is using exclusively `db:setup` and it seems to be working 😄

Reply:
Cool, I'll switch it. :)